.github/workflows/ocwm-issue-collector.yml

```diff
@@ -11,15 +11,15 @@ jobs:
   search_and_add:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Node 18
-      uses: actions/setup-node@v2
+    - uses: actions/checkout@v4
+    - name: Set up Node 20
+      uses: actions/setup-node@v4
       with:
-        node-version: '18'
+        node-version: '20'
     - name: Install dependencies
       run: npm install @octokit/core@5.1.0 
     - name: Adding Issues
-      uses: actions/github-script@v6
+      uses: actions/github-script@v7
```

Comment:
DUPLICATE ACTION CHECKOUT: You use the actions/checkout action twice in your workflow **(@v2 and @v4)**. You should remove one of them to avoid redundancy.

You're specifying the actions/GitHub-script action twice, with different versions (@v6 and @v7). You should use only one version of this action.

Missing Async Function Declaration: Within your JavaScript code block, you use await without an async function. Could you make sure that the script block is wrapped within an async function?

Reply:
Hey @neehhaa06 , there seems to be some misunderstanding. Those lines are not duplicates; rather, they are there to indicate additions and deletions.

Reply:
Hey @neehhaa06 , alok is correct. The red lines indicate code that has been removed, while the green lines represent newly added code. Any lines that remain uncolored signify portions of the codebase that have remained unchanged during the current commit or pull request. Furthermore, GitHub provides the option to switch to a split layout, which allows you to view the file's content before and after the commits, side by side
Regarding your comment about the async function, it's important to note that when the script is used as a module, it allows for top-level await statements. GitHub Actions executes this script as an asynchronous function, enabling the use of await statements within it. This means that we can directly use the await keyword in the module without needing to explicitly define an async function, as the entire module is treated as an asynchronous code block by GitHub Actions.
I hope this clarification helps you understand the underlying concepts more clearly. Feel free to clarify any queries. Thanks!

Reply:
Yeah, Thank you for the clarification @ayushtiwari110